### PR TITLE
Add Force Calls for zkSBT

### PR DESCRIPTION
**Pull request by ferrell-code**
## Description

Add force calls to manually put posts on-chain, must be privileged call.

Add ability for root to set `NextSbtId`.

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [x] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

File: pallets/manta-sbt/src/lib.rs [hunk 1]
```diff
@@ -650,6 +650,91 @@ pub mod pallet {
...
+
+            Self::deposit_event(Event::<T>::ForceToPrivate {
+                asset: asset_id,
+                source: minting_account,
+            });
+            Ok(Pays::No.into())
+        }
+
+        #[pallet::call_index(11)]
+        #[pallet::weight(0)]
```
Comment (ghzlatarev) [inline pallets/manta-sbt/src/lib.rs]: is the weight supposed to be 0?

File: pallets/manta-sbt/src/lib.rs [hunk 2]
```diff
@@ -650,6 +650,91 @@ pub mod pallet {
...
+        #[transactional]
+        #[allow(clippy::too_many_arguments)]
+        pub fn force_mint_sbt_eth(
```
Comment (ghzlatarev) [inline pallets/manta-sbt/src/lib.rs:720]: the force extrinsics are not tested

File: pallets/manta-sbt/src/lib.rs (hunk 1 above)
Comment (ferrell-code) [reply pallets/manta-sbt/src/lib.rs]: No needs benchs

File: pallets/manta-sbt/src/lib.rs (hunk 2 above)
Comment (ferrell-code) [reply pallets/manta-sbt/src/lib.rs:720]: yeah adding tests

File: pallets/manta-sbt/src/lib.rs [hunk 3]
```diff
@@ -650,6 +656,123 @@ pub mod pallet {
...
+            Self::check_post_shape(&post, asset_id)?;
+            let sbt_metadata = MetadataV2::<T::SbtMetadataBound> {
+                mint_id,
+                collection_id: None,
+                item_id: None,
+                extra: Some(metadata),
+            };
+
+            // check that asset id is below max allowed number
+            ensure!(asset_id < MAX_FORCE_ASSET_ID, Error::<T>::TooHighAssetId);
```
Comment (zqhxuyuan) [inline pallets/manta-sbt/src/lib.rs]: Can we make the max asset = the `NextSbtId` when called `setNextSbtId`.

Comment (ferrell-code) [reply pallets/manta-sbt/src/lib.rs]: Yeah we could do that, my thought was just to set `NextSbtId` to MAX_FORCE_ASSET_ID . But we could make it `NextSbtId` instead might be cleaner.

**Review by ghzlatarev — APPROVED** [review]

**Review by Apokalip — APPROVED** [review]

**Review by zqhxuyuan — APPROVED** [review]

**Pull request merged**